### PR TITLE
cla.html: make CLA download links and email address more prominent

### PR DESCRIPTION
**Pull request by mspncp**
See [discussion in #7569](https://github.com/openssl/openssl/pull/7569#issuecomment-435979574).

### CODE REVIEW

File: policies/cla.html
Comment (mattcaswell) [inline policies/cla.html]: This is the old address. New address:

40 East Main Street
Suite 744
Newark, DE  19711
United States

Comment (mspncp) [reply policies/cla.html]: Oh, that's not my fault ;-) I just copy and pasted it:

https://www.openssl.org/policies/openssl_icla.pdf
https://www.openssl.org/policies/openssl_ccla.pdf

But of course I'll fix it in the PR.

**Comment by mspncp** [comment]
The last time I had my own hand-written HTML homepage was somewhere back in the nineties. So the formatting might not be perfect :-). Feel free to change the formatting as you like.

File: policies/cla.html
Comment (mspncp) [reply policies/cla.html]: > This is the old address.

Then it's no wonder that you never received a printed CLA :laughing:

Comment (levitte) [reply policies/cla.html]: I'm not sure anyone has actually tried...  so far, we've received all expected submissions electronically, and *no one* has signalled having tried to send papers and wondering what happened, as far as I can remember.

**Comment by mspncp** [comment]
So what do you think about this change? And shouldn't the postal addresses in the PDF documents be corrected, too?

https://www.openssl.org/policies/openssl_icla.pdf
https://www.openssl.org/policies/openssl_ccla.pdf

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
> And shouldn't the postal addresses in the PDF documents be corrected, too?

Yes. Although I'm not too sure how those PDFs were created in the first place.

**Comment by mspncp** [comment]
> Although I'm not too sure how those PDFs were created in the first place.

If the original documents got lost, I can volunteer to recreate them (using Word, OpenOffice, or LaTeX) for you, if it's not too urgent.

**Comment by mspncp** [comment]
@mattcaswell will you do the merge? I don't think I have push access to the web repo.

**Comment by mattcaswell** [comment]
I think it needs 2 people with commit access to approve first. Unfortunately I don't think your source code commit access counts IIRC.

> If the original documents got lost, I can volunteer to recreate them (using Word, OpenOffice, or LaTeX) for you, if it's not too urgent.

Perhaps @richsalz knows how they were created.

**Comment by mattcaswell** [comment]
Ahh....I found the originals.

**Comment by mspncp** [comment]
ping @openssl/omc for a second review

**Comment by mspncp** [comment]
pling!

**Review by paulidale — APPROVED** [review]

**Comment by mspncp** [comment]
Thank you for reviewing!

@mattcaswell please don't forget to replace the CLA PDFs with the corrected mail address when you merge this pull request :-).

**Comment by mattcaswell** [comment]
> pling!

pling? That's a new one. Is that like this:

https://www.urbandictionary.com/define.php?term=pling

**Comment by mattcaswell** [comment]
I've created a PR to update the addresses in the PDF CLA templates. See #97.

**Comment by mattcaswell** [comment]
Pushed. Thanks!

**Comment by mspncp** [comment]
> > pling!
> 
> pling? That's a new one. Is that like this:
> 
> https://www.urbandictionary.com/define.php?term=pling

Ah no, nice guess, but you are wrong. A 'pling!' is a short wakeup sound like from a bicycle bell. But [listen for yourself](https://securecloud.ncp-e.com/index.php/s/XJazKDtMW8Eqdks). You would probably call it 'ding!'?

(Pun with the similarity 'pling' vs. 'ping' is of course intended ;-)

**Pull request closed without merging**
